src/types/types.js

```diff
@@ -431,8 +431,6 @@ export type EdgeParsedUri = {
 
 export type EdgeEncodeUri = {
   publicAddress: string,
-  segwitAddress?: string,
-  legacyAddress?: string,
```

Comment:
The `encodeUri` method in the core should just take whatever is passed in either of these fields & copy it to `publicAddress`, so plugins can simply ignore these other two & just read `publicAddress`.

Instead of deleting these two from the public Flow types, but instead simply mark them as `// Deprecated`, until our next major version bump, so that this isn't a breaking change.